specs-go/state.go

```diff
@@ -9,7 +9,7 @@ type State struct {
 	// Status is the runtime status of the container.
 	Status string `json:"status"`
 	// Pid is the process ID for the container process.
-	Pid int `json:"pid"`
+	Pid *int `json:"pid"`
```

Comment:
An alternative would be to leave off the pointer and use `omitempty`, since zero isn't a valid PID.  Which way we should go depends on the unresolved #772, but the maintainers may have a one-off opinion on this.